### DISCUSSION TITLE
how to change where prefix's "Documents" points to?

### DISCUSSION THREAD

**Discussion by red-rj** [Heroic-Games-Launcher/HeroicGamesLauncher#4152] [Q&A]
by default it's a symlink to `~/Documents`, but a bunch of games uses Documents to store save data and it clutters the hell of that folder, I'd prefer it not to mix with my other files

specifically, I want the default behavior not to be like this

**Comment by arrowgent** [comment]
that appears to be a wine specification
symlink inside your pfx /../ user folder

https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/1390

https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/3830
